docs/examples/defaultlabelprovider/translations.js

```diff
@@ -26,6 +26,11 @@ const labels = {
     accessibilityHidePasswordLabel: myI18nTranslator('Hide password'),
     accessibilityShowPasswordLabel: myI18nTranslator('Show password'),
   },
+  InfoButton: {
+    accessibilityDefaultTooltipMessage: myI18nTranslator('Click to learn more'),
+    accessibilityDefaultLinkLabel: myI18nTranslator('Learn more'),
+    accessibilityIcon: myI18nTranslator('Question mark icon'),
```

Comment:
We don't need this label anyway, but for future labels: avoid describing the icon itself, and instead focus on what the icon means to sighted users (https://gestalt.pinterest.systems/web/icon#Accessibility)